credentials.yml.example

```diff
@@ -0,0 +1,6 @@
+docker-username: dockerusername
```

Comment:
Is this file just for developer convenience when testing locally?

Reply:
Yes it is - I believe I forgot, I also need to update readme to reflect use of file.